scripts/legislation/africa/namibia_english.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^58. Short title and commencement/,$p' | \
+    sed -n '/^PART I/,$p'
+}
+
+function prefix_article_numbers_with_article_literal {
+  sed -E 's/^([0-9]+\.)/Article \1/'
+}
+
+function replace_parentheses_around_article_delimiters_with_square_brackets {
+  sed -E 's/^\(([A-Za-z0-9]+)\)/[\1]/'
+}
+
+function add_newlines_before_headers_and_articles {
```

Comment:
Please update the name of this function to something more suitable for the function contents.

Reply:
Updating the name of this function results in improper formatting. I have observed that it is not actually necessary to call this function at the end for the proper formatting to result. However, removing this modified add_newlines function will result in improper formatting as well. This makes me quite positive that the apply_common_transformations function is calling on the modified add_newlines function in this local file instead of the one in common.sh. I am not sure why this is as my understanding of how functions are called in bash is quite limited. Please advise on what changes you would like to make with regards to this function.

Reply:
When you change the name of this function, do you also add a call to the new function name to the `preprocess_state_and_language_input_file` function?

Reply:
Yes I do, I place it before `apply_common_transformations_to_stdin`. Do you want me to push what I have and then maybe you can try running it to see if it works on your computer?

Reply:
Push a new revision to this branch (not mainline) with the change. I will then pull this branch and debug. Thanks.

Reply:
After checking out the code and debugging a little, I fully understood what you were trying to address. Please review [my commit](https://github.com/danpdraper/eia/pull/57/commits/7bc19dbd59063d22fa2e1311bb7dbd941edd4aa7). In short, I added functions that move the article title from above the article body to after the article number and suffix the article title with a pipe, move the article title back above the article body after invocation of the `apply_common_transformations_to_stdin` function, and add newlines before the headers embedded in the article bodies.